src/wallets/escrow.js

```diff
@@ -250,6 +290,13 @@ export default class EscrowWallet implements IWallet<AccountData> {
     options: Object
   }): Promise<TxFee> => {
     // send from escrow incurs no tx fees
+    const accountData = this.getAccount().getAccountData()
+    if (accountData.cryptoType === 'bitcoin') {
+      return WalletUtils.getBtcTxFee({
+        value,
+        addressesPool: accountData.hdWalletVariables.addresses
+      })
+    }
     return {
```

Comment:
add else if (["ethereum", "dai"].includes(cryptoType)) to be more specific. More coins will be added in the future.